crates/jstz_cli/src/config.rs

```diff
@@ -1,118 +1,172 @@
 use std::{
-    collections::HashMap,
-    fs,
-    io::{Error, ErrorKind},
+    collections::{hash_map, HashMap},
+    env, fs,
     path::PathBuf,
 };
 
-use anyhow::{anyhow, Result};
+use derive_more::{From, TryInto};
+use jstz_crypto::{public_key::PublicKey, secret_key::SecretKey};
 use jstz_proto::context::account::Address;
 use octez::{OctezClient, OctezNode, OctezRollupNode};
 use serde::{Deserialize, Serialize};
 
 use crate::{
-    account::account::{Account, AliasAccount},
+    error::{bail, user_error, Result},
     jstz::JstzClient,
+    utils::AddressOrAlias,
 };
 
-fn home() -> PathBuf {
-    dirs::home_dir()
-        .expect("Failed to get home directory")
-        .join(".jstz")
+pub fn jstz_home_dir() -> PathBuf {
+    if let Ok(value) = env::var("JSTZ_HOME") {
+        PathBuf::from(value)
+    } else {
+        dirs::home_dir()
+            .expect("Could not find home directory")
+            .join(".jstz")
+    }
+}
+
+// Represents a collection of accounts: users or 'deployments' (aka smart functions)
```

Comment:
How about User&Function?

Reply:
Function could work

Reply:
I went with `SmartFunction`, a bit more specific for our use case